lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -108,13 +109,17 @@ module.exports = {
     artifactFilePaths = artifactFilePaths.concat(
       _.map(layerNames, name => {
         const layerObject = this.serverless.service.getLayer(name);
-
-        if (layerObject.package && layerObject.package.artifact) {
-          return layerObject.package.artifact;
+        if (layerObject.artifactAlreadyUploaded) {
+          this.serverless.cli.log(`Skip uploading ${name}`);
+          return '';
```

Comment:
If we don't want to return anything, but ESLint complains let's return `null`

Reply:
Fixed in https://github.com/serverless/serverless/pull/7680/commits/b8b9d45722f9c1258ef8bf6ffa8d14ac24986e2f